.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+name: Docs to PDF
```

```suggestion
name: Generate PDF
```

We don't have any “docs” here.

And let’s rename the file to something more meaningful, like `pdf.yml`.